app/services/lara_serialization_helper.rb

```diff
@@ -8,6 +8,7 @@ def export(item)
     results = item.export
     results[:type] = item.class.name
     results[:ref_id] = key(item)
+    results[:legacy_ref_id] = item.legacy_ref_id if item.respond_to?(:legacy_ref_id)
```

Comment:
I don't think we want to export the legacy_ref_id. 
If it gets imported into the same or different lara it would create an interactive that looks like it was converted, but really it was just exported and imported.

Reply:
@scytacki OK. I'm confused though on how else to make it accessible to Activity Player. Or am I wrong about AP needing to access this value?

If it does need to stay here, maybe I should've also added the field to the list of excepted fields on line 42 of the import method?

Reply:
After thinking about this some more, it seems like there's no need to make that value accessible to Activity Player... I must've just got mixed up somehow about that. I'll remove that line from lara_serialization_helper.rb.

Reply:
Right, it shouldn't be needed by the AP. It is there to help us when we migrate the learner answers. And it will be useful if we have to track down old data if something goes wrong with the migration.